www/docs/en/dev/guide/next/index.md

```diff
@@ -62,7 +62,7 @@ And many, many, more.
 
 One of the biggest mistakes a new Cordova developer can make is to assume that the performance they get on a desktop machine is the same they will get on a mobile device. While our mobile devices have gotten more powerful every year, they still lack the power and performance of a desktop. Mobile devices typically have much less RAM and a GPU that is a far cry from their desktop (or even laptop) brethren. A full list of tips here would be too much, but here are a few things to keep in mind (with a list of longer resources at the end for further research).
```

Comment:
Yeah this article is not written in a good way, but since we will be deleting this in near future, I only concentrated on removing the bugs and not changing the style of writing.